governance/docs/leadership_selection.md

```diff
@@ -0,0 +1,142 @@
+# Governance: Leadership Selection
+CNCF does not require its hosted projects to follow any specific governance
+model by default. Instead, CNCF specifies that graduated projects need to
+"[e]xplicitly define a project governance and committer process." This varied
+and open governance approach has led to different projects defining what is best
+and optimized for their community.
+
+While there are many options for selecting leaders as part of defining
+governance, the ideal is to have a process that provides a fair and level
+playing field that defines how contributors can become leaders. The governance
+should be documented so that all participants clearly understand the criteria
+and process for moving into and out of leadership positions. Here are more
+details about some of the governance options for CNCF projects.
+
+## Types of Leadership Selection
+
+### Invitation
+
+Leaders may be invited to participate in committees or other leadership roles by
+the people currently in those leadership roles. The Project Management Committee
+for [Vitess](https://github.com/vitessio/vitess/blob/master/GOVERNANCE.md) is by
+invite with a nomination from an existing committee member followed by committee
+approval.
+
+### Committer / Maintainer-based
+
+In many projects, one of the simplest and easiest ways to select leaders is to
+allow the existing committers or maintainers to make the selection. In some
+cases, this might be by invitation from another maintainer, or a person can ask
+to be selected for a role (self-nomination). In
+[Envoy](https://github.com/envoyproxy/envoy/blob/master/GOVERNANCE.md), new
+maintainers are approved via consensus by the group of senior maintainers.
+[Jaeger](https://github.com/jaegertracing/jaeger/blob/master/GOVERNANCE.md) uses
+a nomination process where a new maintainer is nominated by an existing
+maintainer and seconded by two other maintainers.
+
+### Elections and Voting
+
+Elected leadership is often used for the highest levels of leadership in larger
+projects, since defining the election criteria and process can require more
+effort than some of the other options, although there are simpler ways of
+conducting votes. The[ Kubernetes Steering
```

Comment:
```suggestion
conducting votes. The [Kubernetes Steering
```

Reply:
@jberkus Good catch! I just pushed a second commit with that fix.